packages/data-models/deployment.model.ts

```diff
@@ -45,6 +45,15 @@ export interface IDeploymentRuntimeConfig {
     /** prevent user accessing app pages without being logged in. Specified template will be shown until logged in */
     enforceLoginTemplate?: string;
   };
+  campaigns: {
+    /**
+     * Specify whether campaigns notification used by deployment. Default `true`
+     *
+     * NOTE - if disabled functionality will still enable if explicitly called from templates (e.g. `@campaign` authoring)
```

Comment:
Is this still true? It looks like the campaign service won't initialise properly in this case

Reply:
Good catch, thanks. No it is no longer true - I realised after the fact that the task and template-process force init on startup anyways, so no use in attempting to defer until needed. Tidied up with a8fade8cc2b03fa7249750f8f55519d31aef0667